backbone/src/main/java/org/researchstack/backbone/ui/views/SubmitBar.java

```diff
@@ -151,4 +151,30 @@ public View getEditSaveViewActionView() {
     public View getEditSpaceView() {
         return editSpaceView;
     }
+
+    public void updateView(boolean isEditView) {
+        if (isEditView) {
+            getPositiveActionView().setVisibility(GONE);
+            getEditCancelViewActionView().setVisibility(VISIBLE);
+            getEditSaveViewActionView().setVisibility(VISIBLE);
+            getEditSpaceView().setVisibility(VISIBLE);
+        } else {
+            getPositiveActionView().setVisibility(VISIBLE);
+            getEditCancelViewActionView().setVisibility(GONE);
+            getEditSaveViewActionView().setVisibility(GONE);
+            getEditSpaceView().setVisibility(GONE);
+        }
+    }
+
+    public void setEditSaveActionEnabled(int color) {
+        editSaveView.setClickable(true);
+        editSaveView.setTextColor(color);
+
+    }
+
+    public void setEditSaveActionDisabled() {
+        editSaveView.setClickable(false);
+        editSaveView.setTextColor(getResources().getColor(R.color.rsb_submit_disabled));
```

Comment:
I believe `getColor(colorId)` is deprecated. Can you try:

`getColor(R.color.rsb_submit_disabled, getContext().getTheme());`?